src/V3EmitCConstPool.cpp

```diff
@@ -78,6 +78,7 @@ class EmitCConstPool final : public EmitCConstInit {
         m_ofp = newOutCFile();
 
         for (const AstVar* varp : varps) {
+            if (!varp->valuep()) continue;
```

Comment:
Why is a variable in the pool if it does not have a value?  Perhaps should be assert?  Or at least comment why.

Reply:
During implementation I encountered some unexpected segfaults there. It is not expected in normal flow therefore, I'll change it to `UASSERT_OBJ`.